development/progress-bar-app/pages/progress-bar/index.mdx

```diff
@@ -0,0 +1,87 @@
+
+## Table of contents
+
+## Introduction
+
+*`ProgressBar` component enhance websites by providing visual feedback for file uploads, form submissions, content loading, task completion, downloads/updates, and page scrolling. They improve the user experience, engagement, and interface, keeping users informed and motivated in a concise and intuitive manner.
+By incorporating our ProgressBar component, you can elevate your website's user experience to new heights, adding a touch of interactivity and visual appeal. With progress indicators tailored to various scenarios, your website will become more engaging and provide a seamless and delightful user journey.*
+
+## ProgressBar Component Documentation
+The `ProgressBar` component is used to display a progress bar with customizable styles.
+
+## Usage
+To use the `ProgressBar` component, follow these steps:
+
+1. Install the required dependencies:
+```js
+npm install @locoworks/reusejs-react-progress-bar
+```
+or 
+```js
+yarn add @locoworks/reusejs-react-progress-bar
+```
+
+2. Import `ProgressBar`:
+```js
+import { ProgressBar } from "@locoworks/reusejs-react-progress-bar";
+```
+
+## Props
+The ProgressBar component accepts the following props:
+
+- **progressClasses** (optional): A string of additional CSS classes to be applied to the progress bar. These classes will be merged with the default progress bar classes. (Accepts: Tailwind Classes and CSSProperties)
+- **progressContainerClasses** (optional): A string of additional CSS classes to be applied to the progress bar container. These classes will be merged with the default progress bar container classes. (Accepts: Tailwind Classes and CSSProperties)
+- **progressInterval** (required): The interval (in milliseconds) at which the progress should be updated.
+- **running** (optional): A boolean value indicating whether the progress bar is currently running or paused. To start the progress, this value must be set to true. (Default: false)
+- **defaultProgress** (required): The default progress value (between 0 and 100) to be displayed when the progress bar is initially rendered. (Default: undefined)
+- **totalFileSize** (optional): The total file size in bytes. It is used to calculate the progress percentage based on the processed file size. (Default: 0)
+- **processedFileSize** (optional): The processed file size in bytes. It is used to calculate the progress.
+- **radius** (optional): The radius of the circular progress bar (default: 95).
+- **circleRadiousInPercentage** (optional as string): The radius of the circular progress bar in percentage (both radius and circleRadiousInPercentage need to be passed together ex: radius={55} , circleRadiousInPercentage="25%").
+- **enableCircularProgress** (optional as boolean): Boolean value indicating whether to enable the circular progress bar. If true it will render CircularProgress else normal ProgressBar (default: false).
+
+
+**Note :**
```

Comment:
Is this a heading, Should this be shown in table of content? If this is required to be shown in table of content. Change it to ### Note if it is a necessary or use **>** for creating a quote section.